<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Geobase</class>
 <widget class="QDialog" name="Geobase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Геоподосновы</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="title">
        <string>Общее</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>58</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="tbReload">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Обновить</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="resource1.qrc">
             <normaloff>:/some/ico/ico/reload.png</normaloff>:/some/ico/ico/reload.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Настройки</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resource1.qrc">
             <normaloff>:/some/ico/ico/configure.png</normaloff>:/some/ico/ico/configure.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbGeobase">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="title">
        <string>Геоподосновы</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>89</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="tbPlus">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Добавить</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="resource1.qrc">
             <normaloff>:/some/ico/ico/plus.png</normaloff>:/some/ico/ico/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbProps">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Свойства</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="resource1.qrc">
             <normaloff>:/some/ico/ico/edit.png</normaloff>:/some/ico/ico/edit.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbMinus">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Удалить</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="resource1.qrc">
             <normaloff>:/some/ico/ico/minus.png</normaloff>:/some/ico/ico/minus.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbDrawing">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="title">
        <string>Чертежи</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>89</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="tbPlus_2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Добавить</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="resource1.qrc">
             <normaloff>:/some/ico/ico/plus.png</normaloff>:/some/ico/ico/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbProps_2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Свойства</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="resource1.qrc">
             <normaloff>:/some/ico/ico/edit.png</normaloff>:/some/ico/ico/edit.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbMinus_2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Удалить</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="resource1.qrc">
             <normaloff>:/some/ico/ico/minus.png</normaloff>:/some/ico/ico/minus.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGeobaseTree" name="treeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">Объект</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ГИП</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Стадия</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>№ заказа</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>№ уч.</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата получения</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Срок действия</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>kl</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pp</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tp</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>up</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>brd</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>edge</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ko</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>др</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Комментарии</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_geobase">
   <property name="text">
    <string>Добавить геоподоснову...</string>
   </property>
   <property name="toolTip">
    <string>Add geobase</string>
   </property>
  </action>
  <action name="actionProp_geobase">
   <property name="text">
    <string>Свойства геоподосновы...</string>
   </property>
   <property name="toolTip">
    <string>Geobase properties</string>
   </property>
  </action>
  <action name="actionLoad_files">
   <property name="text">
    <string>Добавить чертежи...</string>
   </property>
  </action>
  <action name="actionDel_geobase">
   <property name="text">
    <string>Удалить геоподоснову</string>
   </property>
   <property name="toolTip">
    <string>Remove geobase</string>
   </property>
  </action>
  <action name="actionPlot_prop">
   <property name="text">
    <string>Свойства чертежа...</string>
   </property>
   <property name="toolTip">
    <string>Свойства чертежа геоподосновы</string>
   </property>
  </action>
  <action name="actionDel_files">
   <property name="text">
    <string>Удалить чертеж</string>
   </property>
  </action>
  <action name="actionRecalc_coords">
   <property name="text">
    <string>Пересчитать границы</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>Просмотр в Автокаде</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGeobaseTree</class>
   <extends>QTreeWidget</extends>
   <header>qgeobasetree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbPlus</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_geobase</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbProps</sender>
   <signal>clicked()</signal>
   <receiver>actionProp_geobase</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMinus</sender>
   <signal>clicked()</signal>
   <receiver>actionDel_geobase</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
